### PR TITLE
yacas 1.6.1

### PR DESCRIPTION
**Pull request by ilovezfs**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

depend on cmake and xcode

### CODE REVIEW

**Comment by ilovezfs** [comment]
(or it could go to :skull: since install count is very low)

**Comment by dunn** [comment]
What's the blocker?

File: Formula/yacas.rb [hunk 1]
```diff
@@ -15,18 +15,19 @@ class Yacas < Formula
...
+  def install
+    mkdir "build" do
+      system "cmake", "..", "-G", "Xcode", "-DENABLE_CYACAS_GUI=OFF",
+                            "-DENABLE_CYACAS_KERNEL=OFF", *std_cmake_args
+      xcodebuild "-target", "ALL_BUILD", "-project", "YACAS.xcodeproj",
+                 "-configuration", "Release", "SYMROOT=build/cyacas/libyacas"
+    end
+    bin.install "build/cyacas/libyacas/Release/yacas"
+    lib.install Dir["build/cyacas/libyacas/Release/{libyacas.a,yacas.framework}"]
+    pkgshare.install "scripts"
```
Comment (MikeMcQuaid) [inline Formula/yacas.rb]: Would be good to request a `make install` upstream but otherwise 👍. Sorry for the delay here.

**Review by MikeMcQuaid — APPROVED** [review]

File: Formula/yacas.rb (hunk 1 above)
Comment (ilovezfs) [inline Formula/yacas.rb]: Done: https://github.com/grzegorzmazur/yacas/issues/225.

Comment (MikeMcQuaid) [inline Formula/yacas.rb]: Thanks!

**Comment by ilovezfs** [comment]
@BrewTestBot test this please

**Comment by ilovezfs** [comment]
@BrewTestBot test this please

**Comment by ilovezfs** [comment]
@BrewTestBot test this please

**Comment by ilovezfs** [comment]
@BrewTestBot test this please

**Comment by ilovezfs** [comment]
@BrewTestBot test this please

**Comment by ilovezfs** [comment]
@BrewTestBot test this please

**Comment by ilovezfs** [comment]
@BrewTestBot test this please

**Pull request closed without merging**